Source/WebCore/css/process-css-values.py

```diff
@@ -243,6 +258,18 @@ def _generate_lookup_functions(self, *, to):
                 return string;
             }
 
+            const AtomString& nameStringForSerialization(CSSValueID id)
```

Comment:
Mind adding a link to the spec text that says to lowercase it somewhere - https://drafts.csswg.org/cssom/#serialize-a-css-component-value

Reply:
It's in the commit log but you're right, I should have it as a code comment as well. Will add.